bin/shared/utils.js

```diff
@@ -0,0 +1,3 @@
+export const destructureArgument = argument => argument.split("=");
```

Comment:
Move this file into `/src` folder. I'll explain it later at the end of this review.